From f012d78de4577e9c1b55a63033825d3b13c5d696 Mon Sep 17 00:00:00 2001
From: Peter Hutterer <peter.hutterer@who-t.net>
Date: Wed, 28 Oct 2015 09:18:18 +1000
Subject: [PATCH libinput 1/5] evdev: whitespace fix

Signed-off-by: Peter Hutterer <peter.hutterer@who-t.net>
---
 src/evdev.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/src/evdev.c b/src/evdev.c
index aef0456..45c1b1b 100644
--- a/src/evdev.c
+++ b/src/evdev.c
@@ -297,7 +297,7 @@ evdev_flush_pending_event(struct evdev_device *device, uint64_t time)
 
 		/* Use unaccelerated deltas for pointing stick scroll */
 		if (evdev_post_trackpoint_scroll(device, unaccel, time))
-		    break;
+			break;
 
 		/* Apply pointer acceleration. */
 		accel = filter_dispatch(device->pointer.filter,
-- 
2.4.3

