From 18feed4a85d9553b2d600f487c1f00f346f8fb53 Mon Sep 17 00:00:00 2001
From: Peter Hutterer <peter.hutterer@who-t.net>
Date: Tue, 14 Apr 2015 15:18:19 +1000
Subject: [PATCH libinput] touchpad: Reduce palm detection threshold to 70mm

Signed-off-by: Peter Hutterer <peter.hutterer@who-t.net>
---
 src/evdev-mt-touchpad.c | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/src/evdev-mt-touchpad.c b/src/evdev-mt-touchpad.c
index 68070c2..a663db9 100644
--- a/src/evdev-mt-touchpad.c
+++ b/src/evdev-mt-touchpad.c
@@ -1105,9 +1105,9 @@ tp_init_palmdetect(struct tp_dispatch *tp,
 		if (device->abs.absinfo_x->resolution == 1)
 			return 0;
 
-		/* Enable palm detection on touchpads >= 80 mm. Anything smaller
+		/* Enable palm detection on touchpads >= 70 mm. Anything smaller
 		   probably won't need it, until we find out it does */
-		if (width/device->abs.absinfo_x->resolution < 80)
+		if (width/device->abs.absinfo_x->resolution < 70)
 			return 0;
 	}
 
-- 
2.3.4

