From a899d62c05d322581bd86e2d16be588e4655e1b6 Mon Sep 17 00:00:00 2001
From: Fedora X Ninjas <x@fedoraproject.org>
Date: Fri, 24 Jul 2015 08:35:08 +1000
Subject: [PATCH] Disable thumb detection, it's too aggressive (#1246093)

---
 src/evdev-mt-touchpad.c | 3 +++
 1 file changed, 3 insertions(+)

diff --git a/src/evdev-mt-touchpad.c b/src/evdev-mt-touchpad.c
index cbe8bed..360f930 100644
--- a/src/evdev-mt-touchpad.c
+++ b/src/evdev-mt-touchpad.c
@@ -1507,6 +1507,9 @@ tp_init_thumb(struct tp_dispatch *tp)
 	struct evdev_device *device = tp->device;
 	const struct input_absinfo *abs;
 
+        /* FEDORA: disable thumb detection, too aggressive */
+        return 0;
+
 	abs = libevdev_get_abs_info(device->evdev, ABS_MT_PRESSURE);
 	if (!abs)
 		return 0;
-- 
2.4.3

