From 6dbfd81e26a9adff5b07398ff7e588683e3e6bf6 Mon Sep 17 00:00:00 2001
From: Peter Hutterer <peter.hutterer@who-t.net>
Date: Tue, 23 Jun 2015 12:41:57 +1000
Subject: [PATCH libinput 3/9] evdev: log device's DPI setting if any

Makes debugging things easier.

Signed-off-by: Peter Hutterer <peter.hutterer@who-t.net>
---
 src/evdev.c | 4 ++++
 1 file changed, 4 insertions(+)

diff --git a/src/evdev.c b/src/evdev.c
index 2644b76..67843db 100644
--- a/src/evdev.c
+++ b/src/evdev.c
@@ -1496,6 +1496,10 @@ evdev_read_dpi_prop(struct evdev_device *device)
 					    DEFAULT_MOUSE_DPI);
 			dpi = DEFAULT_MOUSE_DPI;
 		}
+		log_info(libinput,
+			 "Device '%s' set to %d DPI\n",
+			 device->devname,
+			 dpi);
 	}
 
 	return dpi;
-- 
2.4.3

